### DISCUSSION TITLE
Background ignored and can't be transparent when trails enabled?

### DISCUSSION THREAD

**Discussion by roboyeti** [matteobruni/tsparticles#4870] [General]
Hello, didn't think this was an "issue" to file, so thought I would try a discussion first.

I am trying to set background to nothing on the fireworks preset and it works for a second and then goes to black and stays that way.  My requirement for a particle system is that it be able to function as an overlay to DOM elements, and it works fine with confetti.  It appears to be related to trails, as when I was testing an older version (unknowingly), I downloaded the non-minimized code for the preset and fiddled with the trails setting in the JSON.  Is this something that is really required for trails or is it just not a common use case and so hasn't been an issue?  Would it be easy enough to change (I can look into the code and work on it), but if it isn't something that this library can do easily, I will just find a different project to use for my needs.

Thanks!

**Comment by matteobruni** [comment]
Already discussed: [here](https://github.com/matteobruni/tsparticles/discussions/2446), [here](https://github.com/matteobruni/tsparticles/discussions/4244), [here](https://github.com/matteobruni/tsparticles/issues/3495) and other times on Discord. A user on Discord didn't want trails, so I used lines also for fireworks explosions in this sample: https://codepen.io/matteobruni/pen/rNKbrZm?editors=0010, it was fine for him

**Reply by roboyeti** [reply]
Thanks, guess I was not very thorough in my searches.  Sorry about that.  Cool library, for sure, but doesn't sound like it will work for me.  Best of luck with it, it is awesome! :-)
